src/main/java/bio/terra/cli/command/resource/update/BqDataset.java

```diff
@@ -24,6 +25,8 @@ public class BqDataset extends BaseCommand {
   @CommandLine.Mixin ResourceUpdate resourceUpdateOptions;
   @CommandLine.Mixin BqDatasetLifetime bqDatasetLifetimeOptions;
   @CommandLine.Mixin BqDatasetNewIds bqDatasetNewIds;
```

Comment:
Here's the precedent for the "new" we had earlier; there's no real pattern in place. I propose we add "ForUpdate" to the ends of the update-only mixins, or make an `update` package under `options` for consistency. For a later PR.

Reply:
Thanks, I see now